linera-faucet/server/src/lib.rs

```diff
@@ -69,7 +68,7 @@ pub struct ClaimOutcome {
 
 #[derive(Debug, Deserialize, SimpleObject)]
 pub struct Validator {
-    pub name: ValidatorName,
+    pub name: ValidatorPublicKey,
```

Comment:
`public_key: ValidatorPublicKey`

Reply:
Done in [Fix missed renames](https://github.com/linera-io/linera-protocol/pull/3372/commits/1c4535584e4c5794a6c793a09490c90608de4c9a).